db/seeds.rb

```diff
@@ -56,21 +57,61 @@
   end
 end
 
-# Add a note to Patient 2
+# Patients should have names that reflect what we're looking for (e.g. 'Patient With Other Contact' rather than 'Patient 2')
+# Add example of patient with other contact info
+  patient1 = Patient.find_by( name: "Patient 1")
+  patient1.update_attributes( name: "Patient with Other Contact info - 1", other_contact: "Jane Doe", other_phone: "234-456-6789", other_contact_relationship: "Sister" )
+  patient1.save!
+
+# Add example of patient with appointment one week from today && clinic selected
+  patient2 = Patient.find_by( name: "Patient 2")
+  patient2.update_attributes( name: "Patient with clinic and appointment(1wk) - 2",clinic_name: "Sample Clinic 1", appointment_date: Time.now + (7*24*60*60) )
+  patient2.save!
+
+# # Add example of patient with a pledge submitted
+  patient3 = Patient.find_by( name: "Patient 3")
+  pledge = patient3.pledges.new()
+    pledge.created_by = user2
```

Comment:
Not my proudest moment as a PM, but we wound up running out of time before we could really get this part of the object model implemented cleanly. We're using the pregnancy model for this right now, can we move this over to utilize that?